core/domain/stats_services_test.py

```diff
@@ -123,7 +363,7 @@ def test_record_answer(self):
         exp_version = exp.version
 
         for state_name in [first_state_name, second_state_name]:
-            state_answers = stats_services.get_state_answers(
+            state_answers = stats_services_old.get_state_answers(
```

Comment:
Old tests should be moved to the corresponding file (stats_services_old_tests.py)

Reply:
Not relevant anymore.